examples/settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':android-simple'
+include ':android-activity-graphs'
```

Comment:
I find it strange to have an `build.gradle` at the `examples/` level that wouldn't let you build all examples. How about making each `examples/android-*` example an independent Gradle project?